packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.hbs

```diff
@@ -1,60 +1,70 @@
   @header="Deposit Amount"
   @isComplete={{@isComplete}}
 >
-  <section class="card-pay-transaction-amount__body">
-    <header class="card-pay-transaction-amount__body-header">
+  <section class="transaction-amount__body">
+    <header class="transaction-amount__body-header">
       Choose an amount to deposit into the reserve pool
     </header>
-    <CardPay::DepositWorkflow::TransactionAmount::BalanceView
-      @tokenSymbol={{this.currentTokenSymbol}}
-      @tokenIcon={{this.currentTokenDetails.icon}}
-      @tokenDescription={{this.currentTokenDetails.description}}
-      @address={{this.layer1Network.walletInfo.firstAddress}}
-      @balance={{format-token-amount this.currentTokenBalance}}
-      @balanceInUsd={{token-to-usd this.currentTokenSymbol this.currentTokenBalance}}
-    />
-    <div class="card-pay-transaction-amount__input-section">
-      {{#if this.isUnlocked}}
-      <CardPay::LabeledValue
-        @label="Amount to deposit"
-        @icon={{this.currentTokenDetails.icon}}
-        @value={{concat this.amount " " this.currentTokenSymbol}}
-      />
-      {{else}}
+    <div class="transaction-amount__input-section">
       <Boxel::Field
-        @label="Amount to deposit"
-        @fieldMode="edit"
-        class="card-pay-transaction-amount__field"
-        @labelClass="card-pay-transaction-amount__field-label"
+        class="transaction-amount__field"
+        @label="Funding From:"
+        @labelClass="transaction-amount__from-field-label"
       >
-      <div class="card-pay-transaction-amount__amount-field-input-container">
-        {{svg-jar this.currentTokenDetails.icon class="card-pay-transaction-amount__amount-field-currency-icon" role="presentation"}}
-        {{!--
-          .boxel-input is a hack to get the boxel styles.
-          depending on how we plan to handle validation of this input,
-          we may need to change Boxel's input to use a more DDAU pattern
-        --}}
-        <input
-          class="boxel-input card-pay-transaction-amount__amount-field-input"
-          id="deposit-amount-input"
-          value={{this.amount}}
-          {{on "input" this.onInputAmount}}
-          data-test-deposit-amount-input
+        <CardPay::DepositWorkflow::TransactionAmount::BalanceView
+          @wallet={{this.layer1Network.chainName}}
+          @tokenSymbol={{this.currentTokenSymbol}}
+          @tokenIcon={{this.currentTokenDetails.icon}}
+          @address={{this.layer1Network.walletInfo.firstAddress}}
+          @balance={{format-token-amount this.currentTokenBalance}}
+          @balanceInUsd={{token-to-usd this.currentTokenSymbol this.currentTokenBalance}}
+          @isComplete={{@isComplete}}
         />
-        <div class="card-pay-transaction-amount__amount-field-currency-text">
-          {{this.currentTokenSymbol}}
-        </div>
-      </div>
       </Boxel::Field>
+      {{#if (or this.isUnlocking this.isUnlocked)}}
```

Comment:
Since we're using this `or` more than once in the template, consider using a getter with an intention-revealing name instead.